labeled_data/technology/ai/generative_ai/index.yml

```diff
@@ -5,6 +5,7 @@ data:
     - model
     - tool
     - application
+    - infrastructure
   platforms:
```

Comment:
Same problem as `Artificial Intelligence - Application`, if the repos or orgs below can not be set to any sub label, we should add an `other` sub label and move them into that.

Reply:
Yes, it is the same problem.
Right, index should not be frequently changed. However, I'd like to use another sub label `unlabeled` to save them since `other` can be a collection of non primary type items.